src/ui/svg/icons/BicolorProfile.tsx

```diff
@@ -50,3 +50,9 @@ export const BicolorProfile = React.memo(
     opacity: opacity ?? 1,
   }))``
 )
+
+export const Profile = styled(NotMemoizedBicolorProfile).attrs(({ size, theme }) => ({
+  color: theme.colors.black,
+  color2: theme.colors.black,
+  size: size ?? theme.icons.sizes.standard,
+}))``
```

Comment:
Je pense qu'il faut le mémoïser, comme ce qu'on fait avec le BicolorProfile
```suggestion
export const Profile = React.memo(
  styled(NotMemoizedBicolorProfile).attrs(({ size, theme }) => ({
    color: theme.colors.black,
    color2: theme.colors.black,
    size: size ?? theme.icons.sizes.standard,
}))``
)
```

Reply:
J'ai un problème de typing avec l'ajout de React.memo la ou j'utilise l'icon (sans SectionRow) 

```
(property) icon: StyledComponent<React.MemoExoticComponent<StyledComponent<React.FunctionComponent<AccessibleIcon & {
    transform?: string | undefined;
}>, DefaultTheme, {
    color: ColorsEnum;
    color2: ColorsEnum;
    size: string | number;
}, "color" | ... 1 more ... | "color2">>, DefaultTheme, {
    ...;
}, "size">
Impossible d'assigner le type 'StyledComponent<MemoExoticComponent<StyledComponent<FunctionComponent<AccessibleIcon & { transform?: string | undefined; }>, DefaultTheme, { color: ColorsEnum; color2: ColorsEnum; size: string | number; }, "color" | ... 1 more ... | "color2">>, DefaultTheme, { ...; }, "size">' au type 'FunctionComponent<IconInterface> | undefined'.
  Impossible d'assigner le type 'StyledComponent<MemoExoticComponent<StyledComponent<FunctionComponent<AccessibleIcon & { transform?: string | undefined; }>, DefaultTheme, { color: ColorsEnum; color2: ColorsEnum; size: string | number; }, "color" | ... 1 more ... | "color2">>, DefaultTheme, { ...; }, "size">' au type 'FunctionComponent<IconInterface>'.
    Les types de la propriété 'propTypes' sont incompatibles.
      Impossible d'assigner le type 'WeakValidationMap<Omit<Omit<StyledComponentPropsWithAs<string | ComponentType<any>, DefaultTheme, { color: ColorsEnum; color2: ColorsEnum; size: string | number; }, "color" | ... 1 more ... | "color2", string | ComponentType<...>, string | ComponentType<...>> & { ...; }, "size"> & Partial<...>, "theme"> & { ...; }> ...' au type 'WeakValidationMap<IconInterface> | undefined'.
        Le type 'WeakValidationMap<Omit<Omit<StyledComponentPropsWithAs<string | ComponentType<any>, DefaultTheme, { color: ColorsEnum; color2: ColorsEnum; size: string | number; }, "color" | ... 1 more ... | "color2", string | ComponentType<...>, string | ComponentType<...>> & { ...; }, "size"> & Partial<...>, "theme"> & { ...; }>' n'a aucune propriété en commun avec le type 'WeakValidationMap<IconInterface>'.ts(2322)
SectionRowContent.tsx(15, 3): Le type attendu provient de la propriété 'icon', qui est déclarée ici sur le type 'IntrinsicAttributes & SectionRowProps'
```

Je ne sais pas trop comment faire a part ajouter un `as FunctionComponent<IconInterface>` à l'icon